docs/source/data-types/decimal.md

```diff
@@ -1,5 +1,14 @@
 # Decimal
-`Decimal` is represented as [`bigdecimal::BigDecimal`](https://docs.rs/bigdecimal/0.2.0/bigdecimal/struct.BigDecimal.html)
+`Decimal` is represented as `value::CqlDecimal` or [`bigdecimal::BigDecimal`](https://docs.rs/bigdecimal/latest/bigdecimal/struct.BigDecimal.html)
+
+## bigdecimal::BigDecimal
+
+To make use of `bigdecimal::Bigdecimal` type, user should enable `bigdecimal-04` crate feature.
+
+## value::CqlDecimal
```

Comment:
Perhaps we should have separate examples for `BigDecimal` and `CqlDecimal`? We already have separate examples for each type for `time.md`, `date.md` and `timestamp.md` (but it looks like we didn't do it for `varint.md`).